src/hyper-loader/Elements.res

```diff
@@ -1175,7 +1175,15 @@ let make = (
                       mountedIframeRef->Window.iframePostMessage(msg)
                       resolve()
                     })
-                    ->catch(_ => resolve())
+                    ->catch(err => {
+                      logger.setLogError(
+                        ~value=err->formatException->JSON.stringify,
+                        ~eventName=SAMSUNG_PAY_FLOW,
+                        ~paymentMethod="SAMSUNG_PAY",
+                        ~logType=ERROR,
+                      )
+                      resolve()
+                    })
```

Comment:
```suggestion
                    ->catch(err => {
                           let value =
                        [("Samsung Pay Not Ready", err->formatException)]
                        ->Dict.fromArray
                        ->JSON.Encode.object
                        ->JSON.stringify

                      logger.setLogError(
                        ~value,
                        ~eventName=SAMSUNG_PAY_FLOW,
                        ~paymentMethod="SAMSUNG_PAY",
                        ~logType=ERROR,
                      )
                      resolve()
                    })
```

Reply:
Can we label the log value, it will help us in debugging which part of the source code triggered the log.